lib/cErrors.ml

```diff
@@ -46,7 +46,7 @@ exception Timeout = Control.Timeout
 let where = function
 | None -> mt ()
 | Some s ->
-  if !Flags.debug then str "in " ++ str s ++ str ":" ++ spc () else mt ()
+  str "in " ++ str s ++ str ":" ++ spc ()
```

Comment:
I thought this was only for anomalies but it's actually also for user_err (breaking the test suite output tests).

Not sure what to do
- just remove the header for user_err? It's never been useful for me, and many don't have one 
- remove the header for both user_err and anomaly?
- revert and have some reference to break the cerrors/cdebug dependency cycle? (cdebug can produce errors and owns Flags.debug)

Reply:
I dunno, the reference seems a bit ugly, I guess the problem is that we likely shouldn't do the parsing at that level. IMHO always showing the header when we have an error, of just removing it should be fine.